src/docs/markdown/caddyfile/options.md

```diff
@@ -802,6 +802,8 @@ The included [`http_redirect`](/docs/json/apps/http/servers/listener_wrappers/ht
 }
 ```
 
+###### `proxy_protocol`
```

Comment:
For consistency I think we should add a `http_redirect` heading above, and maybe adjust it to not phrase it like "also" and "included" etc

Reply:
Done